modules/nw-egress-ips-about.adoc

```diff
@@ -188,7 +188,7 @@ For users who want an egress IP and traffic to be routed over a particular inter
 
 * The Egress IP must be IPv4. IPv6 is currently unsupported.
 
-* IP forwarding must be enabled for the network interface. To enable IP forwarding, you can set `ipForwarding: Global` in your `config.yaml` file, for example:
+* IP forwarding must be enabled for the network interface. To enable IP forwarding, you can set `ipForwarding: Global` in your Cluster Network Operator configuration object. For example:
```

Comment:
It seems still not cleared how to update it. Here is where to update this add value. 
oc edit network.operator
# ....
spec:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  defaultNetwork:
    ovnKubernetesConfig:
      gatewayConfig:
        ipForwarding: Global
#....